src/xmipp/external/libsvm/svm.cpp

```diff
@@ -2411,45 +2443,55 @@ void svm_cross_validation(const svm_problem *prob, const svm_parameter *param, i
 			++k;
 		}
 		struct svm_model *submodel = svm_train(&subprob,param);
-		if(param->probability && 
+		if(param->probability &&
 		   (param->svm_type == C_SVC || param->svm_type == NU_SVC))
 		{
 			double *prob_estimates=Malloc(double,svm_get_nr_class(submodel));
 			for(j=begin;j<end;j++)
 				target[perm[j]] = svm_predict_probability(submodel,prob->x[perm[j]],prob_estimates);
-			free(prob_estimates);			
+			free(prob_estimates);
 		}
 		else
 			for(j=begin;j<end;j++)
 				target[perm[j]] = svm_predict(submodel,prob->x[perm[j]]);
 		svm_free_and_destroy_model(&submodel);
 		free(subprob.x);
 		free(subprob.y);
-	}		
+	}
 	free(fold_start);
-	free(perm);	
+	free(perm);
 }
 
-#ifdef UNUSED // detected as unused 29.6.2018
+
 int svm_get_svm_type(const svm_model *model)
 {
 	return model->param.svm_type;
 }
-#endif
 
 int svm_get_nr_class(const svm_model *model)
 {
 	return model->nr_class;
 }
 
-#ifdef UNUSED // detected as unused 29.6.2018
 void svm_get_labels(const svm_model *model, int* label)
 {
 	if (model->label != NULL)
 		for(int i=0;i<model->nr_class;i++)
 			label[i] = model->label[i];
 }
 
+void svm_get_sv_indices(const svm_model *model, int* indices)
+{
+	if (model->sv_indices != NULL)
```

Comment:
Unmodified to avoid having two criteria in the same script: NULL (deprecated) and nullptr. Remember that svm.cpp is a library that we replicate. I think that the less we touch the better